recipes-domu/domu-image-weston/domu-image-weston.bbappend

```diff
@@ -11,9 +11,9 @@ XT_QUIRK_BB_ADD_LAYER += "meta-xt-prod-extra"
 ################################################################################
 # Renesas R-Car
 ################################################################################
-SRC_URI_rcar = "repo://github.com/xen-troops/manifests;protocol=https;branch=vgpu-dev;scmdata=keep"
+SRC_URI_rcar = "repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_devel/domu.xml;scmdata=keep"
 XT_QUIRK_PATCH_SRC_URI_rcar = "file://${S}/meta-renesas/meta-rcar-gen3/docs/sample/patch/patch-for-linaro-gcc/0001-rcar-gen3-add-readme-for-building-with-Linaro-Gcc.patch;patchdir=meta-renesas"
-XT_BB_LAYERS_FILE_rcar = "meta-rcar-gen3/doc/bblayers.conf"
+XT_BB_LAYERS_FILE_rcar = "meta-xt-prod-extra/conf/bblayers.conf"
```

Comment:
I guess it would be enough to brake the syntax and see if bitbake failed to parse it.